src/encoded/schemas/atac_frip_quality_metric.json

```diff
@@ -0,0 +1,83 @@
+{
+    "title": "ATAC-seq FRiP quality metric",
+    "description": "Schema for reporting ATAC-seq fraction of reads in peaks quality metrics",
+    "id": "/profiles/atac_frip_quality_metric.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["step_run","quality_metric_of", "award", "lab"],
+    "additionalProperties": false,
+    "identifyingProperties": ["uuid"],
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "quality_metric.json#/properties" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/submitted"},
+        { "$ref": "mixins.json#/aliases"},
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/standard_status"},
+        { "$ref": "mixins.json#/assay" }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "FRiP": {
+            "title": "FRiP score of peaks",
+            "description": "Fraction of reads in the peak file",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 1
+        },
+        "pr1_FRiP": {
+            "title": "FRiP score of pseudoreplicate 1",
+            "description": "Fraction of reads in the peak file of pseudoreplicate 1.",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 1
+        },
+        "pr2_FRiP": {
+            "title": "FRiP score of pseudoreplicate 2",
+            "description": "Fraction of reads in the peak file of pseudoreplicate 2.",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 1
+        },
+        "pr1-pr2_overlap_FRiP": {
+            "title": "FRiP of pseudoreplicate 1 and 2 overlap peaks",
+            "description": "FRiP score of overlap peaks between pseudoreplicate 1 and pseudoreplicate 2",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 1
+        },
+        "pr1-pr2_idr_FRiP": {
+            "title": "FRiP of pseudoreplicate 1 and 2 IDR peaks",
+            "description": "FRiP score of IDR peaks between pseudoreplicate 1 and pseudoreplicate 2",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 1
+        },
+        "rep-rep_overlap_FRiP": {
+            "title": "FRiP of replicate 1 and 2 overlap peaks",
+            "description": "FRiP score of overlap peaks between replicate 1 and replicate 2",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 1
+        },
+        "rep-rep_idr_FRiP": {
+            "title": "FRiP of replicate 1 and 2 IDR peaks",
+            "description": "FRiP score of IDR peaks between replicate 1 and replicate 2",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 1
```

Comment:
if I understand correctly the "rep-rep_idr_FRiP" is going to be associated with 2 files (peaks from rep1 and rep2)? If that is the case, maybe we can give a less concrete title (since the replicates are going to be numbered differently on the portal). Something like "title": "FRiP of replicates IDR peaks", similarly "FRiP of replicates overlap peaks" ? In description, you can avoid numbering the replicates as well.